frappe/printing/page/print/print.js

```diff
@@ -412,6 +412,12 @@ frappe.ui.form.PrintView = class {
 			<link href="${frappe.urllib.get_base_url()}/assets/css/printview.css" rel="stylesheet">`
 		);
 
+		if (out.has_rtl){
```

Comment:
You can use `frappe.utils.is_rtl()`, no need to get this server side

Reply:
`frappe.utils.is_rtl()` is returning true/false based on `frappe.boot.lang` but we need print formats `language`

Reply:
Since we're setting the language client side anyway and passing it server side, instead of returning it from the server side again let's write an `is_rtl` function in print.js itself.